readme.md

```diff
@@ -0,0 +1,178 @@
+<a name="readme-top"></a>
+
+<div align="center">
+   <br/>
+
+  <h3><b>Expense App</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 About the Project ](#-about-the-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 About the Project <a name="about-project"></a>
+
+**Expense App** is an transaction tracker app containing data about transactions based on categories. A user can register/login and create transactions to track them.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Databases</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Contains data about categories**
+- **Contains data about transactions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Link to Live demo]()
+- [Quick Overview]()
```

Comment:
Hello @obibaadoma,

Fantastic work on your project! 👏 I appreciate the effort you've put into it. However, there are a couple of important steps outlined in the project requirements that are yet to be completed:

1. **Video Presentation:**
   - Make sure to record a video presenting your project, following the specified guidelines. This should include a clear project description, a demo of its features, and a highlight of an interesting piece of code. Please provide the link to this video in the README file.

2. **Deployment:**
   - Deploy your application using Render, Netlify, or GitHub Pages. Confirm that both the home and detail pages work as expected. Also, include the link to the deployed project in the README.

Addressing these requirements will significantly improve the overall presentation of your project and offer a more comprehensive understanding for reviewers and potential users.

If you have any questions or need assistance, feel free to reach out. Keep up the excellent work, and I'm eager to see your project in its full glory!

Best regards,
Temitope